exception-handling/practice-exception-handling.md

```diff
@@ -0,0 +1,424 @@
+# Practice: Exception Handling
+
+## Directions
+
+Complete all questions below.
+
+## Practice
+
+<!-- Question 1 -->
+
+<!-- prettier-ignore-start -->
+### !challenge
+
+* type: code-snippet
+* language: python3.6
+* id: XwJz6l
+* title: Exception Handling
+
+### !question
+
+Given these tests, implement the function that will pass. Assume that the only invalid, claimed code school name is `"Ada Developers Academy"`.
```

Comment:
Possible rephrase since we often say that we write code to make tests pass.
```suggestion
Given these tests, provide the function body that will make them pass. Assume that the only invalid, claimed code school name is `"Ada Developers Academy"`.
```

Reply:
heads up, i just wrote like a dozen questions that mirror this wording LOL